.github/workflows/build_and_deploy.yml

```diff
@@ -0,0 +1,104 @@
+name: Mac and Windows Builds
+
+on:
+  push:
+    branches:
+      - '*'
+    tags:
+      - '*'
+
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build-and-deploy:
+    name: Build and Deploy
+    strategy:
+      matrix:
+        os: [windows-latest, macos-latest]
+        python-version: ['3.5', '3.6', '3.7']  # We build 3.5 on Azure Pipelines
+        architecture: ['x86', 'x64']
+        exclude:
+          # Don't build 32-bit on Mac
+          - os: macos-latest
+            architecture: x86
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      # This LOOKS like it is checking out 'master', but it is using the 'master' version of the checkout action
+      # It is actually checking out the most recent version on this branch
+      - name: Checkout
+        uses: actions/checkout@master
+
+      # Visual Studio 2019 (windows-latest) doesn't include these files for some reason??
+      # Even stranger, it really only affects Python 3.5, but we run it across all versions for consistency
+      # https://social.msdn.microsoft.com/Forums/vstudio/en-US/06675e6d-9a0b-46f5-8de0-10237fab1ece/link-error-lnk1158?forum=vcgeneral
+      - name: Fix Windows
+        if: matrix.os == 'windows-latest'
+        run: |
+          Copy-Item "C:\Program Files (x86)\Windows Kits\10\bin\10.0.18362.0\x86\rc.exe" -Destination "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN"
+          Copy-Item "C:\Program Files (x86)\Windows Kits\10\bin\10.0.18362.0\x86\rcdll.dll" -Destination "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN"
+        shell: powershell
+
+      - name: Setting up Python
+        uses: actions/setup-python@v1.1.1
+        with:
+          python-version: ${{ matrix.python-version }}
+          architecture: ${{ matrix.architecture }}
+
+      - name: Updating pip
+        run: python -m pip install --upgrade pip
+        shell: bash
+
+      - name: Installing requirements
+        run: |
+          pip install -r requirements.txt
+          pip install -r build_tools/build_requirements.txt
+        shell: bash
+
+      - name: Building wheel
+        run: make bdist_wheel
+        shell: bash
+
+      - name: Installing generated wheel
+        run: pip install --pre --no-index --find-links dist/ pmdarima
+        shell: bash
+
+      - name: Running unit tests
+        run: |
+          if [ "${{ matrix.os }}" == "macos-latest" ]; then
+            export PMD_MPL_BACKEND=TkAgg
```

Comment:
@charlesdrotar are we using this anymore? I can't remember. If not, we should eventually yank that out of the package itself.

Reply:
Looks like we are using it in [visualization.py](https://github.com/alkaline-ml/pmdarima/blob/master/pmdarima/utils/visualization.py#L21-L23). Strangely the comment says something about Travis builds in particular and `mpl` seems to be in all the visualization functions so we would need to investigate if we should or shouldn't do a clean yank.

Reply:
That said assuming we don't use the environment variable and we aren't using the debug flag in [matplotlib.py's get_compatible_pyplot()](https://github.com/alkaline-ml/pmdarima/blob/master/pmdarima/compat/matplotlib.py#L17), we are basically just doing a normal import of matplotlib.

Reply:
Travis is where we used to build mac, so it probably should have said mac machine. I'll let more knowledgable people decide if we should yank it or not

Reply:
Aaron is correct. This is from the days of yore when we tested viz utils by comparing to PNG files. It can probably be removed at some later point.